Pcap++/header/RawSocketDevice.h

```diff
@@ -86,26 +86,28 @@ namespace pcpp
 		 * @param[in] blocking Indicates whether to run in blocking or non-blocking mode. Default value is blocking
 		 * @param[in] timeout When in blocking mode, specifies the timeout [in seconds] to wait for a packet. If timeout expired
 		 * and no packets were captured the method will return RawSocketDevice#RecvTimeout. Zero or negative values mean no
-		 * timeout. The default value is no timeout
+		 * timeout. The default value is no timeout. The timeout precision is in milliseconds, for example a timeout of 0.123
```

Comment:
Maybe we should mention that the timeout is in seconds but with precision in ms?

Reply:
Note that L87 already mentions that.